src/api/searchApi.ts

```diff
@@ -611,3 +611,40 @@ export const fetchCustomerResults = async (params: FetchCustomerResultsParams, s
 
   return getCustomerResults.data;
 };
+
+export enum UserResultParam {
+  Status = 'status',
+}
+
+export interface FetchUserResultsParams
```

Comment:
Denne er nå identisk med `FetchCustomerResultsParams` over, så må vurdere hva som er mest hensiktsmessig her. Metoden under er også veldig rar nå ved at den defaulter til å søke på relevans, som ikke støttes en gang for dette søket.